unseal_test.go

```diff
@@ -244,7 +250,22 @@ func TestUnsealErrorHandling(t *testing.T) {
 		defer closeTPM(t, tpm)
 
 		err := run(t, tpm, func(keyFile, policyUpdateFile string) {
-			if err := UpdateKeyPCRProtectionPolicy(tpm, keyFile, policyUpdateFile, getTestPCRProfile()); err != nil {
+			src, err := os.Open(keyFile)
+			if err != nil {
+				t.Fatalf("Open failed: %v", err)
+			}
+			defer src.Close()
+
+			newKeyFile := filepath.Dir(keyFile) + "/newkeydata"
+			dst, err := os.OpenFile(newKeyFile, os.O_WRONLY|os.O_CREATE, 0644)
```

Comment:
Is this code strictly for test suite use? I suspect so. If not, this should probably use a more-`mkstemp(3)`-ish approach, including `O_EXCL` and `O_NOFOLLOW` to reduce the risk of unexpected hardlinks and symlinks.

Reply:
Yeah, this is just test code.